lib/matplotlib/pyplot.py

```diff
@@ -1500,65 +1500,59 @@ def axis(*v, **kwargs):
 
 def xlabel(s, *args, **kwargs):
     """
-    Set the *x* axis label of the current axis.
+    Set the x-axis label of the current axes.
 
-    Default override is::
-
-      override = {
-          'fontsize'            : 'small',
-          'verticalalignment'   : 'top',
-          'horizontalalignment' : 'center'
-          }
+    Call signature::
 
-    .. seealso::
+        xlabel(label, fontdict=None, labelpad=None, **kwargs)
 
-        :func:`~matplotlib.pyplot.text`
-            For information on how override and the optional args work
+    This is just a shortcut for calling `.set_xlabel` on the current axes.
+    See there for a full parameter description.
     """
     return gca().set_xlabel(s, *args, **kwargs)
 
 
 def ylabel(s, *args, **kwargs):
     """
-    Set the *y* axis label of the current axis.
+    Set the y-axis label of the current axes.
 
-    Defaults override is::
-
-        override = {
-           'fontsize'            : 'small',
-           'verticalalignment'   : 'center',
-           'horizontalalignment' : 'right',
-           'rotation'='vertical' : }
+    Call signature::
 
-    .. seealso::
+        ylabel(label, fontdict=None, labelpad=None, **kwargs)
 
-        :func:`~matplotlib.pyplot.text`
-            For information on how override and the optional args
-            work.
+    This is just a shortcut for calling `.set_ylabel` on the current axes.
+    See there for a full parameter description.
     """
     return gca().set_ylabel(s, *args, **kwargs)
 
 
 def xlim(*args, **kwargs):
     """
-    Get or set the *x* limits of the current axes.
+    Get or set the x limits of the current axes.
 
-    ::
+    Call signatures::
 
-      xmin, xmax = xlim()   # return the current xlim
-      xlim( (xmin, xmax) )  # set the xlim to xmin, xmax
-      xlim( xmin, xmax )    # set the xlim to xmin, xmax
+        xmin, xmax = xlim()  # return the current xlim
+        xlim((xmin, xmax))   # set the xlim to xmin, xmax
+        xlim(xmin, xmax)     # set the xlim to xmin, xmax
 
-    If you do not specify args, you can pass the xmin and xmax as
-    kwargs, e.g.::
+    If you do not specify args, you can alternatively pass *xmin* or *xmax* as
+    kwargs, i.e.::
 
-      xlim(xmax=3) # adjust the max leaving min unchanged
-      xlim(xmin=1) # adjust the min leaving max unchanged
+        xlim(xmax=3)  # adjust the max leaving min unchanged
+        xlim(xmin=1)  # adjust the min leaving max unchanged
 
     Setting limits turns autoscaling off for the x-axis.
 
-    The new axis limits are returned as a length 2 tuple.
+    Returns
+    -------
+    xmin, xmax
+        A tuple of the new x-axis limits.
 
+    Notes
+    -----
+    This is just a shortcut for calling `~.Axes.get_xlim` or `~.Axes.set_xlim`
```

Comment:
Same as set_xlabel/set_ylable comment. Calling this function with no arguments (e.g. `xlim()`) is the pyplot equivalent of calling `~.Axes.get_xlim` on the current axis. Calling this function with arguments (e.g. xlim(xmin=3, xmax=5) is the pyplot equivalent of calling `~.Axes.set_xlim` on the current axis.